### PR TITLE
Display TX error message

### PR DESCRIPTION
**Pull request by csillag**
We can finally do this:

image

### CODE REVIEW

**Comment by csillag** [comment]
The backend PR we are waiting for is: https://github.com/oasisprotocol/oasis-indexer/pull/347

File: src/app/pages/TransactionDetailPage/index.tsx [hunk 1]
```diff
@@ -64,6 +65,19 @@ const StyledAlert = styled(Alert)(() => ({
   marginBottom: '1em',
 }))
 
+const ErrorBox = styled(Box)(() => ({
+  display: 'flex',
+  justifyContent: 'center',
+  alignItems: 'center',
+  minHeight: '28px',
+  fontSize: '15px',
```
Comment (buberdds) [inline src/app/pages/TransactionDetailPage/index.tsx:73]: wrong size ?

**Review by buberdds — APPROVED** [review]

File: src/app/pages/TransactionDetailPage/index.tsx [hunk 2]
```diff
@@ -131,8 +145,13 @@ export const TransactionDetailView: FC<{
           </dd>
 
           <dt>{t('common.status')}</dt>
-          <dd>
+          <dd style={{ flexWrap: 'wrap', gap: '10px' }}>
             <TransactionStatusIcon success={transaction.success} withText={true} />
+            {transaction.error && (
+              <ErrorBox>
+                {transaction.error.message} (error code {transaction.error.code})
```
Comment (buberdds) [inline src/app/pages/TransactionDetailPage/index.tsx:152]: let's move "error code" label to i18 next

File: src/app/pages/TransactionDetailPage/index.tsx (hunk 1 above)
Comment (buberdds) [reply src/app/pages/TransactionDetailPage/index.tsx:73]: In this case can you double check with Don, please?
Screenshot 2023-03-17 at 09 21 34

Comment (csillag) [reply src/app/pages/TransactionDetailPage/index.tsx:73]: Interesting, I definitely see 15px.

image

File: src/app/pages/TransactionDetailPage/index.tsx (hunk 2 above)
Comment (csillag) [reply src/app/pages/TransactionDetailPage/index.tsx:152]: Done

File: src/app/pages/TransactionDetailPage/index.tsx (hunk 1 above)
Comment (csillag) [reply src/app/pages/TransactionDetailPage/index.tsx:73]: Waiting for the 15px vs 12px decision from @donouwens ...

Comment (csillag) [reply src/app/pages/TransactionDetailPage/index.tsx]: 12px it is!

**Pull request merged**